src/Infrastructure/Twig/Extension/ClassConstantFetcher.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Twig\Extension;
+
+use ReflectionClass;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class ClassConstantFetcher extends AbstractExtension
+{
+    /**
+     * @return array<\Twig\TwigFunction>
+     */
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('class_const', [$this, 'getClassConstant']),
```

Comment:
Why it is decided to use custom function instead of built-in https://twig.symfony.com/doc/2.x/functions/constant.html?